resources/public/api/conf/3.0/scenarios/retrieve_property.md

```diff
@@ -55,5 +55,9 @@
             <td><p>NOT_UK_PROPERTY</p></td>
             <td><p>Simulates the scenario where the calculationId is not a UK property.</p></td>
         </tr>
+        <tr>
```

Comment:
Maybe its this one you need to update ?
resources/public/api/conf/3.0/scenarios/retrieve_foreign_property_bsas.md